### PR TITLE
Feat/1 hello pond

### LINKED ISSUE #1: Create hello pond app
- [x] Init project
- [x] Define idea
- [x] Get example data from github
- [x] Try out all important endpoints from github

### PR DESCRIPTION
**Pull request by Lancear**
Closes #1 

I did this with mr @scarcheek on discord :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+
+- Take PR + commit data from Github
+- Compare commit trees
+- Parse commit files using treesitter
+- Compare ASTs from files to build a better diff
+- Show how the files are related, diagram
+
+## Tech Stack:
+
+- Language: JavaScript ☜(ﾟヮﾟ ☜)
```
Comment (Lancear) [inline README.md:15]: <3

File: info/blob.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (scarcheek) [inline info/blob.json:1]: blob

**Review by scarcheek — APPROVED** [review]
Wow it feels like I was talking with you while you were making that, very well done 👍

File: info/blob.json (hunk 2 above)
Comment (Lancear) [reply info/blob.json:1]: ༼ つ ◕_◕ ༽つ

**Pull request merged**